os/net.h

```diff
@@ -17,10 +17,21 @@
 #include <winsock2.h>
 #include <ws2tcpip.h>
 
+// Standard C includes
+#include <cwchar>
+
 // Macros defined in Linux but not Windows
 #define MSG_NOSIGNAL 0
```

Comment:
I think defining something that is an actual flag on one platform as 0 on another is a little confusing. As you only use it once maybe this would be nicer:
```c++
// Turn of broken pipe signalling (which abruptly kills application) on POSIX
#ifdef _WIN32
const int sendFlags = 0;
#else
const int sendFlags = MSG_NOSIGNAL;
#endif
::send(...);
```